.circleci/config.yml

```diff
@@ -0,0 +1,147 @@
+# alberto CI
+# The Lifted Initiative
+version: 2.1
+
+orbs:
+  node: circleci/node@5.0.2
+
+jobs:
+  test-build:
+    parameters:
+      node_version:
+        type: string
+    executor:
+      name: node/default
+      tag: << parameters.node_version >>
+    steps:
+      - checkout
+      - run:
+          name: Node & NPM version
+          command: |
+            node --version
+            npm --version
+      - install-packages
+      - run:
+          name: Run tests
+          command: npm run test
+      - run:
+          # node 17+ workaround https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported
+          name: set openssl-legacy-provider in NODE_OPTIONS
+          command: |
+              if [ "<< parameters.node_version >>" == "current" ]
+              then
+                echo 'export NODE_OPTIONS=--openssl-legacy-provider' >> $BASH_ENV
+              fi
```

Comment:
Please use a `when` directive instead of doing the check here.

Reply:
when directive is now used.